### PR TITLE
Polish translation - updates

### PR DESCRIPTION
**Pull request by Etharr**

### CODE REVIEW

File: resources/language/Polish/strings.po [hunk 1]
```diff
@@ -128,6 +180,36 @@ msgctxt "#32160"
...
+msgctxt "#32167"
+msgid "Delete lyrics"
+msgstr "Usuń tekst utworu"
+
+# empty strings from id 32168 to 32198
+# Skin labels
 msgctxt "#32199"
 msgid "CU LRC Lyrics"
-msgstr "CU LRC Lyrics"
+msgstr "Tekst utworu"
```
Comment (ronie) [inline resources/language/Polish/strings.po]: please don't change this one.
there is no need to translate the name of the addon.

**Comment by ronie** [comment]
thx! looks good, apart from my comment.
if you can fix that one, i'll merge the pull request.

**Comment by Etharr** [comment]
Ok. I've fixed this.

**Comment by ronie** [comment]
thank you :)

**Pull request merged**